### DISCUSSION TITLE
Easy Backup access?

### DISCUSSION THREAD

**Discussion by Rudiberto** [kimai/kimai#5121] [Questions & Answers] [answered]
I saw these happen on my life access_log:
Can anybody explain, why this direction accesses my Kimai installation, while I'm calling "Easy Backup-Page" first time today inside my Kimai-installation? After that, it did not happen again.
```
159.223.3.117 - - [20/Oct/2024:11:21:13 +0200] "GET / HTTP/1.1" 302 306 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36"
159.223.3.117 - - [20/Oct/2024:11:21:14 +0200] "GET /en/homepage HTTP/1.1" 302 378 "https://my_ip/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36"
159.223.3.117 - - [20/Oct/2024:11:21:14 +0200] "GET /en/login HTTP/1.1" 200 1819 "https://my_ip/en/homepage" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36"
```
 I'm not using the "English" access and 159.223.3.117 is not my external IP, it belongs to:
 OrgName:        DigitalOcean, LLC

All other accesses are calling my hostname, but this one calls the IP.
Am I admitting my IP to DigitalOcean by using Easy Backup? 
Can anybody tell me, if that happens to him/her when accessing the first time (a day) to Easy Backup?

**Comment by kevinpapst** [comment]
@mxgross wanna chim in?

**Comment by mxgross** [comment]
Hi,
The only thing I could imagin adhoc would be the github traffic detection showing which servers are pulling the repository/code. But could this be related to Digital Ocean? And I am not sure if this would happen once a day as you are surely not pulling/fetching every day.
Maybe the ‚git‘ command which is shown in the health check?

EasyBackup has no requests to external sources, nor does it list servers using it, nor collect any information about them.

Maybe you can try installing another Kimai Plugin and see if something simular happens.

**Comment by kevinpapst** [comment]
I would say this is your browser or a browser plugin.

**Comment by Rudiberto** [comment, marked as answer]
I really believe that it has been a coincidence. I activated my access_log just a few minutes before. Since nothing happened I clicked around a little bit and just as I clicked EasyBackup there where these three unknown access intents.
The following minutes I could only see accesses of some employees so I opened these question. 
But since then, there have been several intents, with always the same pattern: three times the same addresses from the same IP - China, GB, USA and so on.
So I apologize for my false alarm. Will try to block these kind of attacks with nginx filters.

**Reply by kevinpapst** [reply]
There is a fail2ban Kimai plugin, that might help to reduce the noise.
